tests/pos/alpha.scala

```diff
@@ -0,0 +1,9 @@
+import annotation.alpha
+
+object Test {
+
+  def foo() = 1
+
+  @alpha("bar") def foo(x: Int) = 2
```

Comment:
I'd like to see a pos test like the following:
```scala
def foo(x: Any): Int = 1
@alpha("bar") def foo[A](x: A): Int = 2
```
Without `@alpha`, those overloads would clash after erasure. But with `@alpha`, they should be fine.